install.sh

```
@@ -16,9 +16,36 @@ WHITE='\033[1;37m'
+VERSION="3.5.1"
 CHECKSUM_FILE="$SCRIPT_DIR/Installers/.checksums.sha256"
 
+# --- 3. SETTINGS & CONFIGURATION ---
+SETTINGS_FILE="$SCRIPT_DIR/settings.conf"
+
+load_settings() {
+    AUTO_UPDATE_CHECK="true"
+    CONFIRM_UPDATES_ON_STARTUP="false"
+    
+    if [[ -f "$SETTINGS_FILE" ]]; then
+        source "$SETTINGS_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)